sep490-enterprise/src/main/resources/application.yml

```
@@ -62,8 +62,8 @@ logging:
   payos:
-    client_id: ${PAYOS_CLIENT_ID:not_a_real_key_to_prevent_null}
-    api_key: ${PAYOS_API_KEY:not_a_real_key_to_prevent_null}
-    checksum_key: ${PAYOS_CHECKSUM_KEY:not_a_real_key_to_prevent_null}
-    returnPath: /success-payment
-    cancelPath: /fail-payment
\ No newline at end of file
+    client_id: ${PAYOS_CLIENT_ID:989e999d-e44e-4d81-854a-ae23fee6de8e}
+    api_key: ${PAYOS_API_KEY:7c3bac52-a3f5-48cf-863b-00c7365fedd1}
+    checksum_key: ${PAYOS_CHECKSUM_KEY:ad2a9afa614f577269e94f02f715894ade2eaa6ed81604a36ef54ecd8ccf2bd6}
+    returnPath: /enterprise/payment
+    cancelPath: /enterprise/payment
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)